checkstyle.xml

```diff
@@ -0,0 +1,352 @@
+<?xml version="1.0"?>
```

Comment:
will delete this file, as we are using the jar directly and `/google_checks.xml` from it for configuration.

Reply:
No need to delete this file, as earlier we are using `/google_checks.xml` but it is not possible to update configuration in this file and we need error to be failed, so we have a copy of this file as `checkstyle.xml`